plotting/plottingUtils/inputManager.cpp

```diff
@@ -0,0 +1,632 @@
+#include "inputManager.h"
+
+// EM: will move this somewhere more sensible
+#define __BAD_FLOAT__ -999.999
+
```

Comment:
Two underscoreso are reserved for the compiler's internal. It is safer to avoid double underscores